### PR TITLE
Nz 11 dashboard entity name

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Add entity name to dashboard reports
- Add entity name to dashboard reports exports
- Improve export filenames
- Tweak EntityView template

---

### Screenshots:
See issue

### CODE REVIEW

File: packages/lesmis/src/components/Tabs.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const TabBarInnerContainer = styled(MuiContainer)`
 
 export const TabBar = ({ children, ...props }) => (
   <TabBarOuterContainer {...props}>
-    <TabBarInnerContainer>{children}</TabBarInnerContainer>
+    <TabBarInnerContainer maxWidth="xl">{children}</TabBarInnerContainer>
```
Comment (tcaiger) [inline packages/lesmis/src/components/Tabs.js:31]: Just making this the same max width as the rest of the templates

File: packages/lesmis/src/views/VitalsView.js [hunk 2]
```diff
@@ -194,12 +194,23 @@ const Container = styled(MuiContainer)`
 `;
 
 const PartnersContainer = styled.div`
+  position: relative;
```
Comment (tcaiger) [inline packages/lesmis/src/views/VitalsView.js:197]: Make the container appear white right to the edge of the page.

File: packages/lesmis/src/components/DashboardReportModal.js [hunk 3]
```diff
@@ -169,7 +171,7 @@ export const DashboardReportModal = () => {
     >
       <DialogHeader
         handleClose={handleClose}
-        title={config?.dashboardName ? config?.dashboardName : 'Loading...'}
+        title={config?.dashboardName ? modalTitle : 'Loading...'}
```
Comment (edmofro) [inline packages/lesmis/src/components/DashboardReportModal.js:174]: I reckon the null check and Loading switch could happen up when the const is created

**Review by edmofro — APPROVED** [review]
Looks good! One very minor and non-required suggestion

**Review by edmofro — APPROVED** [review]
Even better!

**Pull request merged**